arai60/06-07_Stack/06_20_Valid Parentheses/level_2.py

```diff
@@ -0,0 +1,14 @@
+class Solution:
+    def isValid(self, s: str) -> bool:
+        symbols = ["dummy"]
+        mapping = {")": "(", "]": "[", "}": "{"}
+        for i_char in s:
+            if i_char == "(" or i_char == "[" or i_char == "{":
+                symbols.append(i_char)
+            else:
+                close_symbol = symbols.pop()
```

Comment:
elseのなか命名が逆になってると思います。

```suggestion
                open_symbol = symbols.pop()
```